### PR TITLE
Create testing-push-notifications.md

### PR DESCRIPTION
**Pull request by ecempehlivan**

### CODE REVIEW

File: checklists/testing-push-notifications.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+Take care of the modals and try to test each screen. 
+
+- [ ] Check the notification while app is on background  
+Notification wording might be changed while app is on background. Check the wording if it is same with related Jira issue or not.
+
+- [ ] Check the notification is still coming while app is closed  
+
+- [ ] Do all test on both IOS and Android devices 
+
+- [ ] Try to test all different notifications
```
Comment (alpcanaydin) [inline checklists/testing-push-notifications.md]: Maybe we can add testing of deep linking. @ecempehlivan 

```suggestion
- [ ] Try to test all different notifications

- [ ] Test if deep linking works correctly
```

File: checklists/testing-push-notifications.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+---
+
+- [ ] Test the notification while app is on foreground  
+Carefully test all the wordings and emojis.
+
+- [ ] Test the notification in the different screens of the app  
+Take care of the modals and try to test each screen. 
+
+- [ ] Check the notification while app is on background  
+Notification wording might be changed while app is on background. Check the wording if it is same with related Jira issue or not.
```
Comment (akemalakcay) [inline checklists/testing-push-notifications.md]:
```suggestion
Notification wording might change while the app is running on background. Check the wording whether it's the same as related Jira issue or not.
```

File: checklists/testing-push-notifications.md [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+
+- [ ] Check the notification is still coming while app is closed
```
Comment (akemalakcay) [inline checklists/testing-push-notifications.md]:
```suggestion
- [ ] Check to see if the user still gets the expected notification even when the app is closed  
```

File: checklists/testing-push-notifications.md [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+
+- [ ] Test the notification in the different screens of the app  
+Take care of the modals and try to test each screen. 
+
+- [ ] Check the notification while app is on background  
+Notification wording might be changed while app is on background. Check the wording if it is same with related Jira issue or not.
+
+- [ ] Check the notification is still coming while app is closed  
+
+- [ ] Do all test on both IOS and Android devices
```
Comment (akemalakcay) [inline checklists/testing-push-notifications.md]:
```suggestion
- [ ] Apply the tests on both iOS and Android devices  
```

**Pull request merged**